.github/workflows/run-tests.yml

```diff
@@ -60,12 +60,12 @@ jobs:
       - name: Test
         run: |
           cd tests
-          pytest -k "p2p" 
+          pytest -k "p2p"
 
   codecov_in_develop_mode:
 
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 15
```

Comment:
This is because we need to create many DHT instances (=> P2P instances) for averagers in tests. Another solution to consider is making the number of instances smaller.